lib/matplotlib/pyplot.py

```diff
@@ -1017,9 +1013,6 @@ def subplots(nrows=1, ncols=1, sharex=False, sharey=False, squeeze=True,
           always a 2D array containing Axes instances, even if it ends up
           being 1x1.
 
-    num : int or str, optional, default: None
```

Comment:
This is implicitly included in `**fig_kw` and not worth mentioning explicitly here.